snap/snapcraft.yaml

```diff
@@ -0,0 +1,148 @@
+# yamllint disable rule:line-length
+name: postgresql
+base: core22
+version: '14.15'
+summary: PostgreSQL in a snap.
+description: |
+  PostgreSQL is a free and open-source relational database management
+  system emphasizing extensibility and SQL compliance.
+
+architectures:
+  - build-on: [amd64]
+    build-for: [amd64]
+  - build-on: [arm64]
+    build-for: [arm64]
+grade: stable # must be 'stable' to release into candidate/stable channels
+confinement: strict # use 'strict' once you have the right plugs and slots
+
+system-usernames:
+  snap_daemon: shared
+
+plugs:
+  shared-memory:
+    private: true
+
+layout:
+  /usr/lib/python3/dist-packages:
+    bind: $SNAP/usr/lib/python3/dist-packages
+  /etc/perl:
+    bind: $SNAP/etc/perl
+  /usr/local/lib/x86_64-linux-gnu/perl:
+    bind: $SNAP/usr/local/lib/x86_64-linux-gnu/perl
+  /usr/local/share/perl:
+    bind: $SNAP/usr/local/share/perl
+  /usr/lib/x86_64-linux-gnu/perl5:
+    bind: $SNAP/usr/lib/x86_64-linux-gnu/perl5
+  /usr/share/perl5:
+    bind: $SNAP/usr/share/perl5
+  /usr/lib/x86_64-linux-gnu/perl-base:
+    bind: $SNAP/usr/lib/x86_64-linux-gnu/perl-base
+  /usr/lib/x86_64-linux-gnu/perl:
+    bind: $SNAP/usr/lib/x86_64-linux-gnu/perl
+  /usr/share/perl:
+    bind: $SNAP/usr/share/perl
+  /usr/local/lib/site_perl:
+    bind: $SNAP/usr/local/lib/site_perl
+  /usr/lib/postgresql/14:
+    bind: $SNAP/usr/lib/postgresql/14
+  /var/cache/postgresql:
+    bind: $SNAP_COMMON/var/cache/postgresql
+
+# TODO: Add postgresql log path once configured
```

Comment:
Can we address this quickly? Can be followup PR.

Reply:
Looks like a stale comment. The Postgresql are the only thing in the plug.